docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -247,25 +277,62 @@ When linking to another dashboard that uses template variables, select variable
 
 If you want to add all of the current dashboard's variables to the URL, then use `${__all_variables}`.
 
-## Add a data link
+## Add data links or actions {#add-a-data-link}
+
+The following tasks describe how to configure data links and actions.
+
+{{< tabs >}}
+{{< tab-content name="Add data links" >}}
+To add a data link, follow these steps:
 
 1. Navigate to the panel to which you want to add the data link.
 1. Hover over any part of the panel to display the menu icon in the upper-right corner.
 1. Click the menu icon and select **Edit** to open the panel editor.
-1. In the panel edit pane, scroll down to the **Data links** section and expand it.
+1. Scroll down to the **Data links and actions** section and expand it.
 1. Click **+ Add link**.
-1. In the dialog box that opens, enter a **Title**. This is a human-readable label for the link displayed in the UI. This is a required field.
-1. Enter the **URL** or variable to which you want to link. This is a required field.
+1. In the dialog box that opens, enter a **Title**.
+
+   This is a human-readable label for the link displayed in the UI. This is a required field.
+
+1. Enter the **URL** to which you want to link.
 
-   To add a data link variable, click in the **URL** field and enter `$` or press Ctrl+Space or Cmd+Space to see a list of available variables.
+   To add a data link variable, click in the **URL** field and enter `$` or press Ctrl+Space or Cmd+Space to see a list of available variables. This is a required field.
 
 1. If you want the link to open in a new tab, toggle the **Open in a new tab** switch.
-1. If you want the data link to open with a single click, toggle the **One click** switch.
+1. If you want the data link to open with a single click on the visualization, toggle the **One click** switch.
 
    Only one data link can have **One click** enabled at a time. **One click** is only supported for some visualizations.
 
 1. Click **Save** to save changes and close the dialog box.
 1. Click **Save dashboard**.
 1. Click **Back to dashboard** and then **Exit edit**.
 
-If you add multiple data links, you can control the order in which they appear in the visualization. To do this, click and drag the data link to the desired position.
+   {{< /tab-content >}}
+   {{< tab-content name="Add actions" >}}
+   To add an action, by follow these steps:
+
+1. Navigate to the panel to which you want to add the action.
+1. Hover over any part of the panel to display the menu icon in the upper-right corner.
+1. Click the menu icon and select **Edit** to open the panel editor.
+1. Scroll down to the **Data links and actions** section and expand it.
+1. Click **+ Add action**.
```

Comment:
..if available?

Reply:
I think it would be odd to add that in the task, but I'll add a note to reiterate at the top of the task!